app/src/Pages/Portal/Contributions/Utils/ContributionsFields.js

```diff
@@ -87,135 +94,176 @@ export const contributionsEmptyState = {
   linkToDocumentation: '',
   notes: '',
 };
-
+// Converts TypeFieldMap to options for a select
+// ie: DataToContributorTypeFieldMap
+// Just a patch DO NOT USE outside of this file
+export const mapToSelectOptions = mapPairs => {
```

Comment:
```suggestion
const mapToSelectOptions = mapPairs => {
```
remove export if designed not to be used outside of this file

Reply:
Still relevant